codalab/lib/fuse.py

```diff
@@ -0,0 +1,978 @@
+# Copyright (c) 2012 Terence Honles <terence@honles.com> (maintainer)
```

Comment:
Can we acknowledge https://github.com/terencehonles/fusepy more directly or depend on it rather than just copying it?  If it's not too much of a pain.